spawn-cli/activator-cli/src/main.rs

```diff
@@ -45,11 +45,20 @@ async fn main() -> Result<(), async_nats::Error> {
 
     let bytes = Bytes::from(buffer);
 
-    let topic = format!("spawn.{:}.actors.actions", system.clone()).into();
+    let topic: String = format!("spawn.{:}.actors.actions", system.clone()).into();
+    println!("Topic -> {:?}", topic);
+
     let headers = async_nats::HeaderMap::new();
-    nc.publish_with_headers(topic, headers, bytes).await?;
 
-    println!("Published {:?}", request);
+    match nc.request_with_headers(topic, headers, bytes).await {
+        Ok(response) => {
+            println!("Requested -> {:?}", request);
```

Comment:
It might be interesting to use a proper logging library. But that doesn't stop us from proceeding with this PR.